app/views/idv/shared/_reset_your_account.html.erb

```diff
@@ -1,10 +1,3 @@
-<hr>
-  <div class="margin-bottom-2 margin-top-2">
-    <div class="right">
-      <%= link_to image_tag(asset_url('carat-right.svg'), size: '10'),
-        return_to_sp_failure_to_proof_path, class: 'bold block usa-button usa-button--unstyled text-decoration-none' %>
```

Comment:
Heh, the build failure caught an interesting issue, which is that the ">" caret was previously a separate link directing to the wrong URL. This should have been `account_reset_request_path`.